tests/clickhouse-test

```diff
@@ -766,6 +780,20 @@ def collect_tests_to_skip(skip_list_path, build_flags):
 
     return result
 
+
+def collect_sequential_list(skip_list_path):
+    if not os.path.exists(skip_list_path):
+        return set([])
+
+    with open(skip_list_path, 'r') as skip_list_file:
+        content = skip_list_file.read()
+        # allows to have comments in skip_list.json
+        skip_dict = json.loads(json_minify(content))
+        if 'sequential' in skip_dict:
+            return skip_dict['sequential']
```

Comment:
The conditions are negative, so the skip list entry should be called 'parallel' -- tests not to run in 'parallel' case. Same as 'unbundled' means the test we don't run for unbundled build, etc.

Reply:
Yes, I've already thought about it.